### ISSUE TITLE
tab transition on High Sierra fails

### ISSUE THREAD

**Issue by DivineDominion** [sindresorhus/Preferences#38]
On High Sierra, I get this when I switch tabs:

```
*** Assertion failure in -[Preferences.PreferencesTabViewController transitionFromViewController:toViewController:options:completionHandler:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1561.60.100/Controllers/NSViewController.m:925
```

Am investigating!

**Comment by DivineDominion** [comment]
Ok, so

- `PreferencesExample` produces the error,
- my app "Move!" produces the error,
- my apps "WordCounter", "TableFlip", and "The Archive" don't

All use the latest commit from master. (`adde4ea09fe9c9d85b74d4b10dc4a59ade559783`)

**Comment by DivineDominion** [comment]
Looks like the cross-fading animation is the culprit. For some reason I seem to have figured this out in some of my apps ... :)

```swift
if #available(macOS 10.14, *) {
    crossfadeTransitions = true
} else {
    crossfadeTransitions = false
}
```

Will open a PR with a fix!
